storage/engine/keys.go

```diff
@@ -155,6 +155,12 @@ func RangeLastVerificationTimestampKey(raftID int64) proto.Key {
 	return MakeRangeIDKey(raftID, KeyLocalRangeLastVerificationTimestampSuffix, proto.Key{})
 }
 
+// RangeTreeNodeKey returns a range-local key for the the range's
+// node in the range tree.
+func RangeTreeNodeKey(raftID int64) proto.Key {
+	return MakeRangeIDKey(raftID, KeyLocalRangeTreeNodeSuffix, proto.Key{})
```

Comment:
I think these are going to have to use MakeRangeKey instead of MakeRangeIDKey to be addressable in transactions (the comment on KeyAddress says that binary tree pointers will use keys instead of IDs).